### ISSUE TITLE
Accessibility: Keyboard support

### ISSUE THREAD

**Issue by derikb** [johnny/jquery-sortable#207]
Not sure I have the skills to contribute code for this (though I'll give it a try), but it would be great for accessibility if there were support to use the keyboard to sort items. You'd have to add the ability to focus on the sortable elements with tabindex=0 and then add some event delegation for keyboard commands to move the selected element. Probably arrow keys to move items up/down/left/right.

**Comment by mgifford** [comment]
Like to say that this is a big issue.

There's a discussion about best practices for adoption within Drupal 8:
https://www.drupal.org/project/drupal/issues/2920006

But lots of people are working on this on various projects. 

By looking at the demo there are a bunch of accessibility problems, but the lack of keyboard controls is an important one. 

https://johnny.github.io/jquery-sortable/
